maven/plugins/vrealize/src/main/java/com/vmware/pscoe/maven/plugins/PushMojo.java

```diff
@@ -54,30 +54,45 @@ public class PushMojo extends AbstractIacMojo {
 	private List<String> filesChanged;
 
 	private static Package packageFromArtifact(Artifact artifact) {
-		return PackageFactory.getInstance(PackageType.fromExtension(artifact.getType()), artifact.getFile(), new MavenArtifactPackageInfoProvider(artifact).getPackageName());
+		return PackageFactory.getInstance(PackageType.fromExtension(artifact.getType()), artifact.getFile(),
+				new MavenArtifactPackageInfoProvider(artifact).getPackageName());
 	}
 
-	private void importArtifacts(Collection<Artifact> allArtifacts) throws MojoExecutionException {
-		Map<String, List<Artifact>> artifactsByType = allArtifacts.stream().collect(Collectors.groupingBy(Artifact::getType));
-		
+	private void importArtifacts(Collection<Artifact> allArtifacts)
+			throws MojoExecutionException, ConfigurationException {
```

Comment:
God, this ConfigurationException seems like a complete mess... Can't we do `ConfigurationException extends RuntimException` and be done with it?